src/main/webapp/repo-bootstrap/global/blueprints/4000_empty/templates/web/entry.ftl

```diff
@@ -1,30 +1,35 @@
-<#import "/templates/system/common/ice.ftl" as studio />
+<#import "/templates/system/common/crafter.ftl" as crafter />
+
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>${model.title_t}</title>
-  <style>
-    html, body {
-      color: #333;
-      height: 100%;
-      background: #f3f3f3;
-      font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-    }
-    main {
-      max-width: 800px;
-      padding: 40px;
-      background: rgba(255,255,255,0.6);
-      border-radius: 20px;
-      margin: 100px auto;
-    }
-  </style>
-</head>
-<body>
-<main>
-    <@studio.h1 $field="title_t">${model.title_t}</@studio.h1>
-    <@studio.tag $field="body_html">${model.body_html}</@studio.tag>
-</main>
-<@studio.initPageBuilder/>
-</body>
+	<head>
+		<meta charset="utf-8">
+		<title>${model.title_t}</title>
+		<style>
+			html, body {
+				color: #333;
+				height: 100%;
+				background: #f3f3f3;
+				font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+			}
+			main {
+				max-width: 800px;
+				padding: 40px;
+				background: rgba(255,255,255,0.6);
+				border-radius: 20px;
+				margin: 100px auto;
+			}
+		</style>
+		<@crafter.head/>
+	</head>
+	<body>
+		<@crafter.body_top/>
+		<main>
+			<@crafter.h1 $field="title_t">${model.title_t}</@crafter.h1>
+			<@crafter.tag $field="body_html">${model.body_html}</@crafter.tag>
+		</main>
+
+		<script src="/static-assets/js/jquery.core.js"></script>
```

Comment:
jquery.js & crafter-yada-yada.js where removed in my PR to empty bp. looks like you favoured "yours" on during merge. there was some content updates too. please merge correctly.

see https://github.com/craftercms/studio/pull/2592/files

Reply:
Updated, removed import